src/presentation/http/router/join.test.ts

```diff
@@ -4,10 +4,45 @@ describe('Join API', () => {
   describe('POST /join/:hash', () => {
     test('Returns 406 when user is already in the team', async () => {
       const hash = 'Hzh2hy4igf';
-      const userId = 2;
+      const userId = 1;
       const accessToken = global.auth(userId);
 
-      const response = await global.api?.fakeRequest({
+      // truncate all tables, which are needed (i guess we should do it afterAll in each test or beforeAll)
+      await global.db.query(`TRUNCATE public.users, public.notes, public.note_settings CASCADE`);
+
+      // restart autoincrement sequences for data to start with id 1
+      await global.db.query(`ALTER sequence users_id_seq RESTART WITH 1`);
```

Comment:
I think these three lines are repeated everywhere, maybe put them somewhere separate?

Reply:
we decided to do beforeEach function to clear data, so that will be done